app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        issues_found = 0
+        if "vaccine" not in visitor:
+            issues_found += 1
+            raise NotVaccinatedError
+        else:
+            if visitor["vaccine"]["expiration_date"] < date.today():
+                issues_found += 1
+                raise OutdatedVaccineError
```

Comment:
The `OutdatedVaccineError` should be raised with a descriptive message. Consider using `raise OutdatedVaccineError('Visitor's vaccine is outdated.')` to comply with the checklist requirement for descriptive error messages.